AsyncDisplayKit/ASTableView.mm

```diff
@@ -126,6 +126,16 @@ @interface ASTableView () <ASRangeControllerDelegate, ASDataControllerSource> {
 
 @implementation ASTableView
 
+void ASPerformBlockWithoutAnimation(BOOL withoutAnimation, void (^block)()) {
+  if (withoutAnimation) {
+    [UIView setAnimationsEnabled:NO];
```

Comment:
For guaranteed correctness, whenever using this pattern you must:
BOOL animationsEnabled = [UIView areAnimationsEnabled];
...
[UIView setAnimationsEnabled:animationsEnabled];